source/_components/point.markdown

```diff
@@ -11,8 +11,9 @@ logo: minut.svg
 ha_category: Hub
```

Comment:
```suggestion
ha_category: 
  - Hub
  - Alarm
  - Binary Sensor
  - Sensor
```

You can merge all information from the `Alarm`, `Binary Sensor` and `Sensor` page and put it on the `Hub` page and then delete the pages.